### ISSUE TITLE
[A11Y] - All Editorial View | Main Dashboard Enhancements

### ISSUE THREAD

**Issue by Tribunal33** [pkp/pkp-lib#12703]
## Issue Description
General use of the dashboard was a bit confusing.  There were too many options.  

Here is one comment that highlights the accessibility issues surrounding the dashboard: 
> "Suggest making clearer what having a number in a red bubble means in the dashboard. ”Allow for filters to be applied directly on the page. ”Add drop down with categories to filters. ”Identify submissions using title rather than author.” “Separate author and title into two columns in tables.” 
 
Key takeaways 
- Indicate what the red bubble means in the dashboard.  Also add a new color to indicated something is new that needs attention.

<img width="162" height="41" alt="Image" src="https://github.com/user-attachments/assets/41150ef6-5522-4612-b0b4-a90a609e3a38" />

- Allow filters to be applied without opening up a panel directly on the page
- add dropdown with available category filters rather than an auto-complete field.
- Currently operates as a combobox but does not meet the criteria ( https://www.w3.org/WAI/ARIA/apg/patterns/combobox/ )

<img width="106" height="18" alt="Image" src="https://github.com/user-attachments/assets/7a290d72-4297-4be9-8ae2-fe0ce4669b3f" />

<img width="91" height="40" alt="Image" src="https://github.com/user-attachments/assets/b9126fc7-9fcd-4de5-b0e6-56427ab69070" />

- Identify submissions using the title rather than the author.
- Separate the author from the title with a new column.

<img width="158" height="122" alt="Image" src="https://github.com/user-attachments/assets/0c9a34b3-9503-4238-88fa-f0beaf5c9fb3" />

- Increase the font size of the submission tables text (adjust from .75rem to 1rem) this also helps the stage color dot size.  Table header contrast ratio can be improved (I checked and it was already 10+ so should be good enough).

<img width="140" height="69" alt="Image" src="https://github.com/user-attachments/assets/58abb954-dab2-481a-b8d6-f7af67660f70" />

## How to reproduce / Current behavior
Navigate to the Editorial dashboard.  But some of these are visible on Author and Reviewer dashboard views as well.

## What application are you using?
 OJS, OMP or OPS version 3.5.0-3

## Stack used 
- Device: Laptop
- OS: Windows 11
- Browser: Chrome
- Version: Latest

## Testing method
Manual, Using screenreader

## Action
1st priority would be to separate the author and submission title.
2nd priotiy would be to identify what the red bubble indicates (might use a legend or hover note)  -- Warning that something is overdue
3rd Priority would be to have the filters be directly applied to the page without opening up side panel and have categories with a dropdown.

## Additional information
// Please add any screenshots, code snippets, or other information we can use to investigate this issue.

### WCAG SC
- 1.4.1 Use of Color
- 1.3.1 Info and Relationships (A)
- 3.3.2 Labels or Instructions (A)
- 4.1.2 Name, Role, Value (A) - combo box issue
